beginner_source/blitz/data_parallel_tutorial.py

```diff
@@ -1,52 +1,50 @@
 """
-Optional: Data Parallelism
+선택 사항: 데이터 병렬 처리 (Data Parallelism)
 ==========================
-**Authors**: `Sung Kim <https://github.com/hunkim>`_ and `Jenny Kang <https://github.com/jennykang>`_
+**글쓴이**: `Sung Kim <https://github.com/hunkim>`_ and `Jenny Kang <https://github.com/jennykang>`_
+**번역**: '정아진 <https://github.com/ajin-jng>'
 
-In this tutorial, we will learn how to use multiple GPUs using ``DataParallel``.
+이 튜토리얼에서는 ``DataParallel`` (데이터 병렬) 을 사용하여 여러 GPU를 사용하는 법을 배우겠습니다.
 
-It's very easy to use GPUs with PyTorch. You can put the model on a GPU:
+PyTorch를 통해 GPU를 사용하는 것은 매우 쉽습니다. 먼저, 모델을 GPU에 넣습니다:
 
 .. code:: python
 
     device = torch.device("cuda:0")
     model.to(device)
 
-Then, you can copy all your tensors to the GPU:
+그 다음으로는 모든 Tensors 를 GPU로 복사합니다:
 
 .. code:: python
 
     mytensor = my_tensor.to(device)
 
-Please note that just calling ``my_tensor.to(device)`` returns a new copy of
-``my_tensor`` on GPU instead of rewriting ``my_tensor``. You need to assign it to
-a new tensor and use that tensor on the GPU.
+''my_tensor.to(device)'' 를 호출 시 에는 ''my_tensor'' 를 다시쓰는 대신 ''my_tensor'' 의 또다른 복사본이 생긴다는 사실을 기억하십시오. 
+당신은 그것을 새로운 tensor 에 소속시키고 GPU에 그 tensor를 써야합니다.
 
-It's natural to execute your forward, backward propagations on multiple GPUs.
-However, Pytorch will only use one GPU by default. You can easily run your
-operations on multiple GPUs by making your model run parallelly using
-``DataParallel``:
+여러 GPU를 톧해 앞과 뒤의 전파를 실행하는 것은 당연한 일 입니다.
```

Comment:
```suggestion
여러 GPU를 톧해 순전파 및 역전파를 실행하는 것은 당연한 일 입니다.
```
forward / backward prop- 이 순전파 / 역전파라고 많이 번역되는것 같습니다.

Reply:
그렇군요, 감사합니다!